### PR TITLE
introduce vm-config package to fix local-npm test

### PR DESCRIPTION
**Pull request by turadg**
## Description

https://github.com/Agoric/agoric-sdk/pull/8102 broke the some agoric-cli integration tests. 
 (https://github.com/Agoric/agoric-sdk/actions/runs/5753924697/job/15598129426 )

https://github.com/Agoric/agoric-sdk/pull/8140 helped but there was one problem left with resolving the configs.

This fixes that by separating configs to their own package.

It also adds a timeout and fixes a benchmark test missed in the 'boot' move.

### Security Considerations

n/a

### Scaling Considerations

n/a

### Documentation Considerations

I don't think these are documented outside the repo.

### Testing Considerations

Improves factoring

### Upgrade Considerations

The configs are only used in bootstrap, not upgrade.

### CODE REVIEW

File: .github/workflows/integration.yml
Comment (mhofman) [inline .github/workflows/integration.yml]: Let's move this at the `job` level instead of this individual step

**Review by mhofman — DISMISSED** [review]
Just a review of the GH action change

File: packages/vm-config/README.md
Comment (dckc) [inline packages/vm-config/README.md]: Plans seem like they belong in issues. Or at least: when we say "we plan to X" I prefer that X is linked to an issue.

And historical info ("Factored out...") belongs in commit messages.

File: packages/vm-config/package.json
Comment (dckc) [inline packages/vm-config/package.json:16]: Our [docs on making a new package](https://github.com/Agoric/agoric-sdk/blob/master/README.md#adding-a-new-package) say:
> populate a new `packages/zoe/package.json`, using other packages as a template

Is there any more to it than that? I'm sometimes overwhelmed wondering which other packages to use as a template.

File: packages/vm-config/README.md
Comment (dckc) [inline packages/vm-config/README.md:7]: This seems like useful justification.

But I wonder to what extent it's true. The configs _do_ reference other packages:

```json
  "coreProposals": [
    "@agoric/vats/scripts/init-core.js",
    "@agoric/vats/scripts/init-network.js",
    {
      "module": "@agoric/inter-protocol/scripts/init-core.js",
```

**Review by dckc — COMMENTED** [review]
I think it's worth waiting on review by @michaelfig for approval.

Meanwhile, some observations / questions.

File: packages/vm-config/README.md
Comment (turadg) [reply packages/vm-config/README.md]: not firm enough plans to merit an issue, imo. if it can't be here I would just drop it. 

I could qualify this with "possible future directions"

File: packages/vm-config/package.json
Comment (turadg) [reply packages/vm-config/package.json:16]: Those docs are a little stale. e.g. we don't have to manually add to workspaces now because it has a globl.
I'll try to update in this PR.

as a template I used `boot/packages.json` (i actually copied the whole dir then cut it down)

File: packages/vm-config/README.md
Comment (turadg) [reply packages/vm-config/README.md:7]: yeah… in a way it's not true. It's only true in the way that you can _get_ the package without any other dependencies.

File: packages/vm-config/README.md
Comment (dckc) [reply packages/vm-config/README.md]: > if it can't be here

It's not a critical thing. I just wonder about it getting out of date. I get the impression that it's likely to be addressed soon anyway.

File: packages/vm-config/package.json
Comment (turadg) [reply packages/vm-config/package.json:16]: separated https://github.com/Agoric/agoric-sdk/pull/8142

File: packages/vm-config/README.md
Comment (turadg) [reply packages/vm-config/README.md:7]: thinking more, this is fully true. the configs have no dependencies. the runtime that wants to build or execute those configs does.

File: packages/inter-protocol/scripts/start-local-chain.sh
Comment (michaelfig) [inline packages/inter-protocol/scripts/start-local-chain.sh:58]: Does this (and the reference in `test-proposal-stuff.js`) mean that `inter-protocol/package.json` should take a dependency on `@agoric/vm-config`?

**Review by michaelfig — APPROVED** [review]
Just one missing dependency from `@agoric/inter-protocol` to `@agoric/vm-config`, but that's easy to fix even if you land this PR first.

File: packages/inter-protocol/scripts/start-local-chain.sh
Comment (turadg) [reply packages/inter-protocol/scripts/start-local-chain.sh:58]: I don't think so, because this is instructing `cosmic-swingset` what to do. That's the package that really has the dependency. Same as how the JSONs in vm-config that specify a slate of packages doesn't itself depend on them (the thing that reads their source does).

Also this script is not long for this world https://github.com/Agoric/agoric-sdk/pull/7914

**Pull request merged**
